### PR TITLE
add link to deprecation warning

### LINKED ISSUE #76684: '"include" is deprecated, use include_tasks/import_tasks instead.' deprecation warning is without link to place explaining how to deal with it
### Summary

I noticed

> [DEPRECATION WARNING]: "include" is deprecated, use include_tasks/import_tasks instead. This feature will be removed in version 2.16.

Sadly, this deprecation warning is without link to documentation explaining how it should be fixed

I went to changelog: but https://github.com/ansible/ansible/blob/devel/changelogs/CHANGELOG.rst is not really helping with finding documentation

searching `import_tasks include_tasks difference ansible` on internet found finally https://serverfault.com/questions/875247/whats-the-difference-between-include-tasks-and-import-tasks

Every single person using Ansible will need to search for about 2 to 10 minutes - multiply it by number of Ansible users and there is quite big waste of time. 

If there is a good reason to deprecate something - please link to documentation explaining how to fix code that worked fine so far.

Or maybe at least mention which version is equivalent to the current behaviour.

(ideally there would be `ansible undeprecate_include` which would edit all instances of `include` to whichever version is matching current one and `ansible undeprecate_include include_tasks` which would use `include_tasks` and `ansible undeprecate_include import_tasks`)

### Issue Type

Documentation Report

### Component Name

ansible (sorry)

### Ansible Version

```console
$ ansible --version

ansible [core 2.12.1]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/mateusz/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  ansible collection location = /home/mateusz/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.8.10 (default, Nov 26 2021, 20:14:08) [GCC 9.3.0]
  jinja version = 2.10.1
  libyaml = True
```

### Configuration

```console
$ ansible-config dump --only-changed

(no visible output)
```

### OS / Environment

Lubuntu 20.04

### Additional Information

No googling needed to find docs, lower risk of finding misleading docs

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

Fixes #76684

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

lib/ansible/playbook/helpers.py
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/playbook/helpers.py [hunk 1]
```diff
@@ -95,6 +95,10 @@ def load_list_of_tasks(ds, play, block=None, role=None, task_include=None, use_h
     from ansible.playbook.role_include import IncludeRole
     from ansible.playbook.handler_task_include import HandlerTaskInclude
     from ansible.template import Templar
+    from ansible.utils.plugin_docs import get_versioned_doclink
+
+# we'll need this down here
+    include_link = get_versioned_doclink('user_guide/playbooks_reuse_includes.html')
```
Comment (bcoca) [inline lib/ansible/playbook/helpers.py]: move to inside the else, since it is the only place where it is used (avoid doing work when not needed)

**Review by bcoca — APPROVED** [review]

**Review by bcoca — CHANGES_REQUESTED** [review]
can we also get a changelog?

**Comment by samccann** [comment]
you want fries with that documentation?  :-)  Yep will do.

**Comment by samccann** [comment]
@bcoca I think this one is ready now...

**Comment by bcoca** [comment]
@samccann still needs changelog, I had missed that, but it was brought up during triage, since this is candidate to backport

**Comment by samccann** [comment]
@bcoca doh!  I created the changelog last week but pushed to wrong branch... fixed now, thanks!

**Review by bcoca — APPROVED** [review]

**Comment by bcoca** [comment]
saw that ... NEVAR happens to me .. no sir .. nope .. much less a dozen times ...

**Pull request merged**